docker/install.sh

```diff
@@ -17,4 +17,4 @@
 sudo chmod -R +x /opt/carma/install
 source /opt/ros/kinetic/setup.bash
 cd ~/
-catkin_make install -DCMAKE_BUILD_TYPE=Release
+colcon build --cmake-target install --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
Did you find that "--cmake-target install" was required?
My understanding was that install should be the default for colcon build so this wouldn't be needed. If this is not needed please remove from all repos.